extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -108,10 +125,15 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     if (this.hasAppendedInnerShoppingTagEl_ || !tagData) {
       return;
     }
+
     this.mutateElement(() => {
+      this.shoppingTagEl_ = renderShoppingTagTemplate(tagData, (tagData) =>
+        this.onClick_(tagData)
+      );
+      this.onRtlStateUpdate_(this.storeService_.get(StateProperty.RTL_STATE));
```

Comment:
These lines can be outside of the mutateElement since: 
- `renderShoppingTagTemplate` does not modify the dom. 
- `onRtlStateUpdate_` uses `mutateElement`.